src/groundlight/images.py

```diff
@@ -49,7 +49,8 @@ def parse_supported_image_types(
         # Already in the right format
         return image
     if isinstance(image, np.ndarray):
-        return BytesIO(jpeg_from_numpy(image, jpeg_quality=jpeg_quality))
+        # Assume it is in BGR format from opencv
```

Comment:
Should we print a warning here the first time a user passes us a numpy array? Esp since this is a breaking change